src/eko/output/struct.py

```diff
@@ -253,7 +254,7 @@ def __post_init__(self):
             value = getattr(self, attr)
             if value is None:
                 continue
-            if isinstance(value, np.ndarray):
+            if isinstance(value, np.ndarray) or isinstance(value, list):
```

Comment:
Most likely you need `list` because you changed above, and from serialization it will be a `dict`, but it is not a bad idea to keep.

However, you don't need two calls to `isinstance`:
```suggestion
            if isinstance(value, (np.ndarray, list)):
```
https://docs.python.org/3/library/functions.html#isinstance

Reply:
Ok thank you :)